### PR TITLE
dx-843 Vipps capture

### PR DESCRIPTION
**Pull request by arebra**
Made a separate Capture section for Vipps, fixed some info that was wrong in other sections.

### CODE REVIEW

File: _includes/vipps-payments-capture.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+```mermaid
+sequenceDiagram
+  activate Merchant
+  Merchant->>-SwedbankPay: POST [Vipps captures]
+  activate SwedbankPay
+  SwedbankPay-->>-Merchant: transaction resource
+```
+
+[operations]: /payments/vipps/other-features#operations
```
Comment (payex-dev) [inline _includes/vipps-payments-capture.md:74]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777&bid=20134207)

File: payments/vipps/capture.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (payex-dev) [inline payments/vipps/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777)

File: _includes/vipps-payments-capture.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+## Capture
```
Comment (payex-dev) [inline _includes/vipps-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777)

File: _includes/payment-order-capture.md [hunk 4]
```diff
@@ -8,12 +8,12 @@
 
 
 Capture can only be done on a payment with a successful authorized transaction,
-and if the authorization was not done on a one-phase payment instrument.
-Examples of one-phase payment instruments are [Swish][swish] and [Vipps][vipps],
- while payment instruments such as [Card Payments][card] are two-phase payments,
- requiring a `Capture` to be performed.
+and if the authorization was not done with a one-phase payment instrument.
+Examples of one-phase payment instruments are [Swish][swish] and
+[Trustly][trustly], while payment instruments such as [Card Payments][card] are
```
Comment (asbjornu) [inline _includes/payment-order-capture.md:13]: Should we perhaps add `Vipps` as an example of two-phase payment instruments as well?

File: _includes/vipps-payments-capture.md (hunk 1 above)
Comment (asbjornu) [reply _includes/vipps-payments-capture.md:74]: Is the `[operations]` link definition in use? If not, remove it?

File: payments/vipps/after-payment.md [hunk 5]
```diff
@@ -293,10 +249,11 @@ documentation_section="vipps" transaction="cancel" %}
 
 ## Create cancellation transaction
 
-A payment may be cancelled if the `rel` `create-cancellation` is available.
-You can only cancel a payment, or part of it, if it has yet to be captured.
-To revert a capture, or part of a capture, you must perform a `reversal`.
-Performing a cancellation will cancel all remaning capture amounts on a payment.
+A payment may be cancelled if the `rel` `create-cancellation` is available. You
```
Comment (asbjornu) [inline payments/vipps/after-payment.md:252]:
```suggestion
A payment may be cancelled if the operation `create-cancellation` is available. You
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
Great stuff!

File: _includes/vipps-payments-capture.md (hunk 1 above)
Comment (arebra) [reply _includes/vipps-payments-capture.md:74]: It is in use, but it seems to be in an include in the Vipps section, and not in the card section where I took it from, so I am trying to fix it now 👍

File: _includes/payment-order-capture.md (hunk 4 above)
Comment (arebra) [reply _includes/payment-order-capture.md:13]: Will do 👍

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777&bid=20134207)

File: _includes/vipps-payments-capture.md (hunk 1 above)
Comment (payex-dev) [inline _includes/vipps-payments-capture.md:74]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777)

File: payments/vipps/capture.md (hunk 2 above)
Comment (payex-dev) [inline payments/vipps/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777)

File: _includes/vipps-payments-capture.md (hunk 3 above)
Comment (payex-dev) [inline _includes/vipps-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777)

File: _includes/vipps-payments-capture.md (hunk 1 above)
Comment (asbjornu) [reply _includes/vipps-payments-capture.md:74]: The `[operations]` link does not seem to be in use.

```suggestion
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/vipps-payments-capture.md (hunk 1 above)
Comment (arebra) [reply _includes/vipps-payments-capture.md:74]: I got it to work now, with the help of @NullableInt 👍

**Review by asbjornu — APPROVED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777&bid=20134207)

**Review by asbjornu — APPROVED** [review]

File: _includes/transaction-list-response.md [hunk 6]
```diff
@@ -86,7 +86,10 @@ Content-Type: application/json
...
+| └─➔&nbsp;`receiptReference`       | `string`  | A unique reference for the transaction. This reference is used as an invoice/receipt number.                                                                                                                 | {% endif %}
 | └─➔&nbsp;`isOperational`          | `bool`    | `true` if the transaction is operational; otherwise `false`.                                                                                                                                                 |
-| └─➔&nbsp;`operations`             | `array`   | The array of operations that are possible to perform on the transaction in its current state.                                                                                                                |
+| └─➔&nbsp;`operations`             | `array`   | The array of [`operations`][operations] that are possible to perform on the transaction in its current state.
```
Comment (payex-dev) [inline _includes/transaction-list-response.md:92]: Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777&bid=20134207)

File: payments/vipps/capture.md (hunk 2 above)
Comment (payex-dev) [inline payments/vipps/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777)

File: _includes/vipps-payments-capture.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
```
Comment (payex-dev) [inline _includes/vipps-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6164777)

**Pull request merged**